docs/en/Authorization.md

```diff
@@ -393,6 +393,39 @@ public override void ConfigureServices(ServiceConfigurationContext context)
 
 This is already done for the startup template integration tests.
 
+### Claims Principal Factory
+
+Abp abstracts the way that authentication creates `ClaimsPrincipal`. You can provide a custom `IAbpClaimsPrincipalContributor` to add additional claims.
+
+Example of add `SocialSecurityNumber` of current user to claims:
+
+```csharp
+public class SocialSecurityNumberClaimsPrincipalContributor : IAbpClaimsPrincipalContributor, ITransientDependency
+{
+    public async Task ContributeAsync(AbpClaimsPrincipalContributorContext context)
+    {
+        var identity = context.ClaimsPrincipal.Identities.FirstOrDefault();
+        if (identity != null)
+        {
+            var currentUser = context.ServiceProvider.GetRequiredService<ICurrentUser>();
+            if (currentUser.Id.HasValue)
+            {
+                var userManager = context.ServiceProvider.GetRequiredService<IdentityUserManager>();
+                var user = await userManager.GetByIdAsync(currentUser.Id.Value);
+                if (user?.SocialSecurityNumber != null)
```

Comment:
It's not possible. `user` is an instance of `IdentityUser` that does NOT have `SocialSecurityNumber`.
It should be something like `user.GetProperty("SocialSecurityNumber", default(string))`.

Reply:
It’s just a example.